docs/large_inputs.md

```diff
@@ -1,4 +1,63 @@
 # Handling large inputs
 
-TODO
+The Variant Transforms pipeline can process hunderds of thousands of files,
+millions of samples, and billions of records. Please consider adjusting the
+default settings as described below when processing any dataset that is more
+than a few gigabytes or contains more than 50,000 files (even if the individual
+files are small).
+
+In addition, it is recommended to run the pipeline with
+`--optimize_for_large_inputs` to process large datasets more efficiently.
+
+## Numer of workers
+
+The default maximum number of workers is 15. You can adjust this by adding
+`--max_num_workers <num workers>` to the pipeline args.
+The actual number of workers will be dynamically adjusted based on Dataflow's
+autoscaling algorithm. If you prefer to not use the autoscaling algorithm, you
+may set `--num_workers <num workers>` to use the exact
+number of workers.
+
+### Adjusting quota
+
+To increase the number of workers, you need to adjust the Compute Engine quota.
+In particular, you need at least `num_workers + 1` IPs, `num_workers + 1` CPUs,
+and `num_workers * 250GB` of disk in the zone you are running the pipeline.
```

Comment:
FYI, In my experience even 250G is not enough. In one run for 1000Genome, I had 32 machines exhausting 13T disk pretty quickly.